content/pt-br/docs/reference/glossary/aggregation-layer.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Camada de Agregação
+id: aggregation-layer
+date: 2018-10-08
+full_link: /pt-br/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation/
+short_description: >
+  A camada de agregação permite que você instale APIs adicionais no estilo Kubernetes em seu cluster.
+
+aka: 
+tags:
+- arquitetura
+- extensão
+- operação
```

Comment:
This change breaks the glossary, as the canonical tags are currently hardcoded. Could you please revert it?

Reply:
Sorry about that. I did before your explanation and forgot to fix.